docs/api/navigators/DrawerNavigator.md

```diff
@@ -128,6 +128,7 @@ const styles = StyleSheet.create({
 
 ### `contentOptions` for `DrawerView.Items`
 
+- `routes` - an array of routes to use in place of `navigation.state.routes`
```

Comment:
I could change the name to items. Though I should note that the actual use case is not passing arbitrary data to `DrawerView.Items`, it's splitting `navigation.state.routes` into groups and passing a subset of `navigation.state.routes` to multiple `DrawerView.Items`.

Very crudely:
```
<DrawerView.Items routes={navigation.state.routes.slice(0, 1)} ... />
{some divider}
<DrawerView.Items routes={navigation.state.routes.slice(1, 2)} ... />
{some heading}
<DrawerView.Items routes={navigation.state.routes.slice(3, 4)} ... />
```

You probably could pass some data not part of navigation.state.routes in. However the implementation is still relies on `navigation.state` for a few things such as what item is focused. So you can't actually pass in 100% custom data and have complete control over the items view.